packages/botonic-react/src/components/message.jsx

```diff
@@ -154,31 +169,56 @@ export const Message = props => {
 
   const isFromUser = () => from === 'user'
   const isFromBot = () => from === 'bot'
+
   const getBgColor = () => {
     if (!blob) return COLORS.TRANSPARENT
-    return isFromUser()
-      ? getThemeProperty('brand.color', COLORS.BOTONIC_BLUE)
-      : COLORS.SEASHELL_WHITE
+    if (isFromUser()) {
+      const color = getThemeProperty('message.user.style.background', undefined)
+      return color || getThemeProperty('brand.color', COLORS.BOTONIC_BLUE)
+    }
+    const color = getThemeProperty('message.bot.style.background', undefined)
+    return color || COLORS.SEASHELL_WHITE
```

Comment:
suggestion: you can use the second parameter of getThemeProperty as a fallback to the default properties. If the property is not found in the theme, then the second parameter will be used.